marketplace/commercelayer-sku/extension.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "commercelayer-sku",
+  "name": "Commerce Layer SKU",
+  "srcdoc": "./build/index.html",
+  "fieldTypes": ["Object"],
+  "parameters": {
+    "instance": [
+      {
+        "id": "clientID",
+        "name": "Client ID",
+        "type": "Symbol",
+        "description": "The Client ID of your application on Commerce Layer"
+      },
+      {
+        "id": "clientSecret",
+        "name": "Client secret",
+        "type": "Symbol",
+        "description": "The Client secret of your application on Commerce Layer"
```

Comment:
Could we add some instructions so this advertises using a read-only-token or similar?

Reply:
I added a bit more context. On the commerce layer site you're able to create an oauth token to be specifically consumed by Contentful. They take care of the permission on their side. There are images of the process in the readme.